src/main/java/io/excaliburfrc/robot/Constants.java

```diff
@@ -30,8 +30,12 @@
   public static class ClimberConstants {
     public static final int FORWARD_CHANNEL = 0;
     public static final int REVERSE_CHANNEL = 1;
-    public static final int CLIMBER_SPARKMAX = 1;
-    public static final int SENSOR_CHANNEL = 1;
+    public static final int LEFT_MOTOR_ID = 51;
+    public static final int RIGHT_MOTOR_ID = 52;
+
+    public static final double kP = 0;
+    public static final double kI = 0;
+    public static final double kD = 0;
```

Comment:
shouldn't these have some sort of initial value?

Reply:
we'll tune them later